specs/ppd-search.spec

```diff
@@ -0,0 +1,222 @@
+# Price Paid Data
+
+Tags: PPD, price paid data
+
+The price paid data search form
+
+## Building search
+
+Tags: Search
+
+* Type "Rose Cottage" into the "Building name or number" field
+* Clicking show results will route User to the search results page
+* Summary will display '9873 matching transactions'
+* Summary will display '62 properties'
+* First result item address displays as 'Rose Cottage, North Road, Wells, BA5 1LA'
+* Reset the form
+
+## Street search
+
+Tags: Search
+
+* Type "Harbour Road" into the "Street" field
+* Clicking show results will route User to the search results page
+* Summary will display '2192 matching transactions'
+* First result item address displays as '73 Harbour Road, Bradford, BD6 3RG'
+* Reset the form
+
+## Town search
+
+Tags: Search
+
+* Type "Plymouth" into the "Town or city" field
+* Clicking show results will route User to the search results page
+* Summary will display '10000 or more matching transactions'
+* Summary will display '60 properties'
+* First result item address displays as '25 Trinity Street, Plymouth, PL1 3FT'
+* Reset the form
+
+## District search
+
+Tags: Search
+
+* Type "City of Westminster" into the "District" field
+* Clicking show results will route User to the search results page
+* Summary will display '10000 or more matching transactions'
+* Summary will display '68 properties'
+* First result item address displays as 'Flat 93, Chiltern Court, Baker Street, London, NW1 5TA'
+* Reset the form
+
+## County search
+
+Tags: Search
+
+* Type "Devon" into the "County" field
+* Clicking show results will route User to the search results page
+* Summary will display 'from 10000 or more matching transactions'
+* Summary will display '57 properties'
+* First result item address displays as 'The Summerhouse, Lyme Regis, DT7 3XU'
+* Reset the form
+
+## Locality search
+
+Tags: Search
+
+* Type "Thurloxton" into the "Locality" field
+* Clicking show results will route User to the search results page
+* Summary will display '71 transactions'
+* First result item address displays as 'Greenway Farm House, Taunton, TA2 8RF'
+* Reset the form
+
+## Postcode search
+
+Tags: Search
+
+* Type "PL6" into the "Postcode" field
+* Clicking show results will route User to the search results page
+* Summary will display '10000 or more matching transactions'
+* Summary will display '61 properties'
+* First result item address displays as 'Rose Cottage, North Road, Wells, BA5 1LA'
+* Reset the form
+
+// NB: full postcode supplied returns 0 results every time. Is that a bug?
```

Comment:
Full postcodes work correctly on the current production app, so if you're getting zero results on the dev app then this is definitely a regression

Reply:
Now recorded as an issue: https://github.com/epimorphics/hmlr-linked-data/issues/60